### ISSUE TITLE
Your input ran out of data; interrupting -> why repeat is not working here !

### ISSUE THREAD

**Issue by tarrade** [tarrade/proj_multilingual_text_classification#45]
I 2020-05-06T12:35:53.789589643Z master-replica-0 2103/2105 [============================>.] - ETA: 8s - loss: 0.2903 - accuracy: 0.8791  master-replica-0 
I 2020-05-06T12:35:53.789877413Z master-replica-0  training set -> batch:2104 loss:0.150290846824646 and acc: 0.8820000290870667 master-replica-0 
I 2020-05-06T12:35:53.789999246Z master-replica-0  master-replica-0 
E 2020-05-06T12:36:17.876522778Z master-replica-0 Your input ran out of data; interrupting training. Make sure that your dataset or generator can generate at least `steps_per_epoch * epochs` batches (in this case, 29 batches). You may need to use the repeat() function when building your dataset. master-replica-0 
  undefined

**Comment by tarrade** [comment]
seems it was an issue on my side using wrong number of events. None is a way to run over all data. It seems that tf.data.dataset need repeat() anyway
